File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/MyPosts/SinglePostFragMyPostActAdm.java
package com.example.adminapp.MyPosts;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.adminapp.R;



public class SinglePostFragMyPostActAdm extends Fragment {



    public SinglePostFragMyPostActAdm() {
        // Required empty public constructor
    }



    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_single_post_frag_my_post_act_adm, container, false);
    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/MyPosts/SinglePostFragMyPostActAdm.kt
package com.example.adminapp.MyPosts

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.adminapp.R

class SinglePostFragMyPostActAdm : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(
            R.layout.fragment_single_post_frag_my_post_act_adm,
            container,
            false
        )
    }
}